src/Symfony/Component/Cache/Adapter/PdoAdapter.php

```diff
@@ -108,9 +108,20 @@ public function createTable()
         $conn = $this->getConnection();
 
         if ($conn instanceof Connection) {
+            $types = array(
+                'mysql'  => 'binary',
+                'sqlite' => 'text',
+                'pgsql'  => 'string',
+                'oci'    => 'string',
+                'sqlsrv' => 'string',
+            );
+            if (!isset($types[$this->driver])) {
+                throw new \DomainException(sprintf('Creating the cache table is currently not implemented for PDO driver "%s".', $this->driver));
+            }
+            
             $schema = new Schema();
             $table = $schema->createTable($this->table);
-            $table->addColumn($this->idCol, 'blob', array('length' => 255));
+            $table->addColumn($this->idCol, $types[$this->driver], array('length' => 255));
```

Comment:
Well, given that works I'd suggest to make the end result the same. Using the initially proposed mapping will result that. Still I don't know why col type text is needed for sqlite, but probably I've got some catching up to do concerning that...

So, I now propose to keep it like my initial proposition. What about you?